app/route_rangers_api/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
is there a separate ticket to integrate this into the existing frontend @lisette-solis? 

from first glance this needs to be plugged into the survey work and pushing a new `index.html` and `main.js` might break things when run from `python -m manage runserver` so my gut says we should not merge this into main b/c that would break production, but if it looks good we can create a new branch to take all of this cool work and plug it into the django system.